client/generated_disk_info.go

```diff
@@ -9,9 +9,9 @@ type DiskInfo struct {
 
 	AllowScheduling bool `json:"allowScheduling,omitempty" yaml:"allow_scheduling,omitempty"`
 
-	Path string `json:"path,omitempty" yaml:"path,omitempty"`
+	Conditions map[string]string `json:"conditions,omitempty" yaml:"conditions,omitempty"`
```

Comment:
This generated wrong. Need to use the same method as `nodeCondition` and `volumeCondition` to generate the `map[string]interface{}` instead of `map[string]string`